update-app-doc.xsl

```diff
@@ -317,7 +348,7 @@ the terms of the GNU Affero General Public License as published by the Free
 Software Foundation, version 3 of the License.
 
 ## Copyright
-Copyright (c) 2017 - 2018 [Radio Bern RaBe](http://www.rabe.ch)
+Copyright (c) 2017 - 2019 [Radio Bern RaBe](http://www.rabe.ch)
```

Comment:
Can we insert the current year automatically instead of having it hard-coded?

Reply:
Actually, that this is much earlier (ie. before the # Usage part) is very much intended.

Reply:
Addressed with cd53272cb442a5636834e478facc0395be82d293